.azure-pipelines.yml

```diff
@@ -105,6 +105,7 @@ jobs:
     vmImage: 'macOS-10.13'
   steps:
   - template: tools/ci/azure/checkout.yml
+  - script: brew install python
```

Comment:
@foolip would you rather we create a new file for this, or put it in `tools/ci/azure/tox_pytest.yml`? (note that `python` here, without a version specifier, is Python 3)

Reply:
I think it's important that we use the system python by default, so in `tools/ci/azure/tox_pytest.yml` would work, or just at the top of the new jobs that would use py3. I'd probably use https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops to vary just the version of Python used.

You could also check if https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/use-python-version?view=azure-devops works on macOS, the "not on Linux or macOS agents" bit isn't entirely clear so it might.

Reply:
This change wasn't needed in the end; the bot already had 3.7 installed, just not 3.6

Reply:
Correct.